src/ScriptEditor/ui/ScriptEditorRoot.tsx

```diff
@@ -813,20 +822,30 @@ export function Root(props: IProps): React.ReactElement {
         ></Box>
 
         <Box display="flex" flexDirection="row" sx={{ m: 1 }} alignItems="center">
-          <Button startIcon={<SettingsIcon />} onClick={() => setOptionsOpen(true)} sx={{ mr: 1 }}>Options</Button>
+          <Button startIcon={<SettingsIcon />} onClick={() => setOptionsOpen(true)} sx={{ mr: 1 }}>
+            Options
+          </Button>
           <Button onClick={beautify}>Beautify</Button>
-          <Button color={updatingRam ? "secondary" : "primary"} sx={{ mx: 1 }} onClick={() => { setRamInfoOpen(true) }}>
+          <Button
+            color={updatingRam ? "secondary" : "primary"}
+            sx={{ mx: 1 }}
+            onClick={() => {
+              setRamInfoOpen(true);
+            }}
+          >
             {ram}
           </Button>
-          <Button onClick={save}>Save (Ctrl/Cmd + s)</Button>
+          <Button sx={{ mr: 1 }} onClick={save}>
+            Save (Ctrl/Cmd + s)
+          </Button>
           <Button onClick={props.router.toTerminal}>Close (Ctrl/Cmd + b)</Button>
           <Typography sx={{ mx: 1 }}>
             {" "}
-            Documentation:{" "}
+            <strong>Documentation:</strong>{" "}
             <Link target="_blank" href="https://bitburner.readthedocs.io/en/latest/index.html">
               Basic
-            </Link>{" "}
-            |
+            </Link>
+            {" | "}
```

Comment:
Cleaned up a bit of spacing here, looks like Prettier did it's thing as well. Let me know and I can yoink these changes from the PR